### DISCUSSION TITLE
Migrating database to another type of database. [Mar 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1535] [Discord Threads]
> 🙋‍♂️ **andrew7771**

**Comment by dstala** [comment]
> 💁 **o1lab**  
#### It not possible as of today.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/954825427552198727)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Reply by igmarketing** [reply]
@dstala @o1lab Is it possible now? 

I'm thinking of moving away of SQLite since my NocoDB instance is slow to load data... I think this could be faster using PostgreSQL.

If I want to move, what's the best approach to not lose my data?
